atoma-confidential/src/service.rs

```diff
@@ -255,7 +258,7 @@ impl AtomaConfidentialComputeService {
             let tdx_quote = get_compute_data_attestation(&public_key_bytes)?;
             let tdx_quote_bytes = tdx_quote.to_bytes();
             match self
-                .sui_client
+                ._sui_client
```

Comment:
why is it `_sui_client` when you are actually using it?

Reply:
Because it is behind a feature

Reply:
Disabled dead code